wls-gui-wahllokalsystem/src/components/common/tables/BaseTableRowManager.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-container>
+    <v-form v-model="isChangeRowCountFormValid">
+      <div class="d-flex">
+        <base-number-input
+          v-model="newRowCount"
+          :rules="validationRules"
+          max-width="15rem"
+          :label="inputFieldLabel"
+        />
+        <v-btn
+          active
+          :disabled="isApplyRowCountDisabled"
+          class="ml-4 mt-3"
+          @click="onChangeRowCountClicked"
+        >
+          {{ applyBtnLabel }}
+        </v-btn>
+      </div>
+    </v-form>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import type { PropType } from "vue";
+
+import { computed, ref } from "vue";
+
+import BaseNumberInput from "@/components/common/inputs/BaseNumberInput.vue";
+import { useRules } from "@/composables/common/rules.ts";
+
+const { required } = useRules();
+
+const newRowCount = defineModel<number | null>({ required: true });
+
+const props = defineProps({
+  currentRowCount: {
+    type: Number,
+    required: true,
+  },
+  rules: {
+    type: Array as PropType<((value: number) => string | boolean)[]>,
+    required: false,
+    default: () => [],
+  },
+  applyBtnLabel: {
+    type: String,
+    required: false,
+    default: "Übernehmen",
+  },
+  inputFieldLabel: {
+    type: String,
+    required: false,
+    default: "Anzahl",
+  },
+});
+
+const emit = defineEmits<{
+  changeRowCountClicked: [newValue: number | null];
+}>();
+
+const isChangeRowCountFormValid = ref<boolean | null>(null);
+
+const isApplyRowCountDisabled = computed(
+  () =>
+    newRowCount.value === null ||
+    isChangeRowCountFormValid.value !== true ||
+    newRowCount.value === props.currentRowCount
+);
+
+const validationRules = computed(() => {
+  return props.rules ? [required, ...props.rules] : [required];
```

Comment:
```suggestion
  return  [required, ...props.rules];
```

sollte auch funktionieren

Reply:
[0be81cb](https://github.com/it-at-m/Wahllokalsystem/pull/2014/commits/0be81cb82e1f69b7abad51474bde1973de151162)